### PR TITLE
#1153 - SFTP Production Setup

### PR DESCRIPTION
**Pull request by jherrflexion**
Terraform and Github Workflow needed to support the production deploy

## Issue
[#1153](https://github.com/CDCgov/trusted-intermediary/issues/1153)

### CODE REVIEW

**Comment by halprin** [comment]
Mind updating the PR description to use the canonical issue URL?

File: operations/environments/prod/main.tf [hunk 1]
```diff
@@ -0,0 +1,33 @@
+terraform {
```
Comment (halprin) [inline operations/environments/prod/main.tf:1]: The folder this file is in needs to be `prd`, not `prod`.

File: operations/environments/prod/main.tf [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      source  = "hashicorp/azurerm"
+      version = "3.113.0"
+    }
+  }
+
+  # Use a remote Terraform state in Azure Storage
+  backend "azurerm" {
+    resource_group_name  = "cdcti-terraform"
+    storage_account_name = "cdctiterraform"
+    container_name       = "tfstate"
```
Comment (halprin) [inline operations/environments/prod/main.tf:13]:
```suggestion
    container_name       = "terraform-state"
```
I think this should be `terraform-state` just like staging.

File: operations/environments/prod/main.tf [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "3.113.0"
+    }
+  }
+
+  # Use a remote Terraform state in Azure Storage
+  backend "azurerm" {
+    resource_group_name  = "cdcti-terraform"
```
Comment (halprin) [inline operations/environments/prod/main.tf:11]:
```suggestion
    resource_group_name  = "csels-rsti-prd-moderate-rg"
```

File: operations/environments/prod/main.tf [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    storage_account_name = "cdctiterraform"
```
Comment (halprin) [inline operations/environments/prod/main.tf:12]:
```suggestion
    storage_account_name = "cdcintermediarytrfrmprd"
```

You'll need to make this storage account too.

**Review by halprin — DISMISSED** [review]
Some things need to change for this to work correctly.  It's been a hot sec since I've truly had to look at this (vacation and all), so if something doesn't make sense, please ask.

**Review by pluckyswan — APPROVED** [review]

**Review by halprin — APPROVED** [review]
🔥

**Pull request merged**